extensions/api/keypair-mgmt-api/src/main/java/org/eclipse/edc/identityhub/api/keypair/v1/KeyPairResourceApi.java

```diff
@@ -82,6 +82,22 @@ public interface KeyPairResourceApi {
     )
     void addKeyPair(String participantId, KeyDescriptor keyDescriptor, boolean makeDefault, SecurityContext securityContext);
 
+
+    @Tag(name = "KeyPairResources Management API")
+    @Operation(description = "Sets a KeyPairResource to the ACTIVE state. Will fail if the current state is anything other than ACTIVE or CREATED.",
+            responses = {
```

Comment:
Could you please add the `participantId` path param for this new API ? See here: https://github.com/eclipse-edc/IdentityHub/pull/285/files